sdk/communication/communication-identity/CHANGELOG.md

```diff
@@ -10,6 +10,19 @@
 
 ### Other Changes
 
+## 1.4.0-beta.1 (2025-03-02-preview)
```

Comment:
This date should be the actual date when we're planning to release the SDK, not the REST API version